explore-analyze/elastic-inference/eis.md

```diff
@@ -0,0 +1,58 @@
+---
+navigation_title: Elastic Inference Service (EIS)
+applies_to:
+  stack: ga 9.0
+  serverless: ga
+---
+
+# Elastic {{infer-cap}} Service [elastic-inference-service-eis]
+
+The Elastic {{infer-cap}} Service (EIS) enables you to leverage AI-powered search as a service without deploying a model in your cluster.
+With EIS, you don't need to manage the infrastructure and resources required for {{ml}} {{infer}} by adding, configuring, and scaling {{ml}} nodes.
+Instead, you can use {{ml}} models for ingest, search, and chat independently of your {{es}} infrastructure.
+
+## AI features powered by EIS [ai-features-powered-by-eis]
+
+* Your Elastic deployment or project comes with a default [`Elastic Managed LLM` connector](https://www.elastic.co/docs/reference/kibana/connectors-kibana/elastic-managed-llm). This connector is used in the AI Assistant, Attack Discovery, Automatic Import and Search Playground.
+
+* {applies_to}`stack: preview 9.1` {applies_to}`serverless: preview` You can use [ELSER](/explore-analyze/machine-learning/nlp/ml-nlp-elser.md) to perform semantic search as a service (ELSER on EIS).
```

Comment:
Do we have another page for ELSER? It could be confusing to link to the ML node docs in a sentence that advertises EIS.